### ISSUE TITLE
Adding login page and adding email verification 

### ISSUE THREAD

**Issue by praveenarjun** [kartikayasijaa/talk-trove#9]
Feature Request: Add Login Page and Email Verification

Description:

1. Login Page:

    Develop a user-friendly login page that allows users to securely access their accounts.
    The login form should include fields for entering the email and password.
    Implement error handling for incorrect login credentials.
    Include options for "Forgot Password" and "Sign Up" to enhance user experience.

2. Email Verification:

    Implement an email verification system to ensure valid email addresses during user registration.
    Upon registration, send a verification email to the user's provided email address.
    The email should contain a unique verification link that the user must click to verify their email.
    Ensure the user cannot access certain features of the application until their email is verified.

Additional Details:

    Integrate the login and email verification features seamlessly with the existing user authentication system.
    Ensure that the login page adheres to the design and styling guidelines of the current application.
    Provide clear and concise feedback messages to users during the login and verification processes.

Please proceed with implementing these features to enhance the security and user experience of our application.

**Comment by praveenarjun** [comment]
pls assign the labels it will be helpfull and motivative

**Comment by Sibam-Paul** [comment]
@praveenarjun  I am mentor of this repo try to raise issues , after get response from the PA , I will assign it

**Comment by praveenarjun** [comment]
OK it's fine I just need a reply thank you for your support

**Comment by Sibam-Paul** [comment]
> OK it's fine I just need a reply thank you for your support

It will take time for PA to add me as collaborator so have patience 
try raising as much issues u can

**Comment by praveenarjun** [comment]
Ok

**Comment by Sibam-Paul** [comment]
@praveenarjun The UI has been assigned by #6 
U can too work on your issue Go ahead

**Comment by Sibam-Paul** [comment]
@praveenarjun Are u working on it any update ??

**Comment by praveenarjun** [comment]
Yes total I will pull request

**Comment by praveenarjun** [comment]
I added a pull request #22 
But what happening In instruction it is specifies not in main but there is not other branch in admin so push in main

**Comment by praveenarjun** [comment]
pls check that is there any issue occured or not

**Comment by Sibam-Paul** [comment]
> I added a pull request #22 But what happening In instruction it is specifies not in main but there is not other branch in admin so push in main

What where ???

**Comment by praveenarjun** [comment]
#22  this is my pull request once check it

**Comment by Sibam-Paul** [comment]
@praveenarjun Yup just sat down to review it

**Comment by Sibam-Paul** [comment]
@praveenarjun Can u please submit a small video for now as there is no discord channel to discuss I need to ask the PA to do that or to add discussion tab to this repo will be helpful

**Comment by praveenarjun** [comment]
give a level for the pull  request  

> @praveenarjun Can u please submit a small video for now as there is no discord channel to discuss I need to ask the PA to do that or to add discussion tab to this repo will be helpful

yes I will but it takes some time nearly a 1hr

**Comment by Sibam-Paul** [comment]
@praveenarjun yeah take ur time bro no rush and Are u working on the UI , will see it

**Comment by praveenarjun** [comment]
I work on UI of mail and verification page I used dom in js so the total html file is in email.js file

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Sibam-Paul** [comment]
@praveenarjun 
![image](https://github.com/user-attachments/assets/08fa6bb6-1d8a-4e80-9180-88c73fe7cde6)
![image](https://github.com/user-attachments/assets/530422ea-330b-41a4-bb2c-685de64984a6)
Can do to the url

**Comment by praveenarjun** [comment]
![Screenshot 2024-10-05 at 1 04 58 AM](https://github.com/user-attachments/assets/ad03ac07-5743-4e54-a1ae-57d11afa06fe)

**Comment by praveenarjun** [comment]
![Screenshot 2024-10-07 at 7 06 15 PM](https://github.com/user-attachments/assets/f4121e88-cae0-4219-a205-f99d6efb0dd9)
here you need to add the mail and password

**Comment by SumitSahu44** [comment]
hey @Sibam-Paul , @kartikayasija if the work is pending so you can assign this to me i can do it under gssoc-ext label
